py3-ansible.yaml

```diff
@@ -0,0 +1,75 @@
+package:
+  name: py3-ansible
+  version: 2.18.0
+  epoch: 0
+  description: Ansible is a radically simple IT automation platform
+  copyright:
+    - license: GPL-3.0
+  dependencies:
+    provider-priority: 0
+
+vars:
+  pypi-package: ansible
+  import: ansible
+
+data:
+  - name: py-versions
+    items:
+      3.10: "310"
+      3.11: "311"
+      3.12: "312"
+      3.13: "300"
+
+environment:
+  contents:
+    packages:
+      - py3-supported-build-base
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: ec78526b976481f3fcdf91a904eeaa68b89ccdea
+      repository: https://github.com/ansible/ansible
+      tag: v${{package.version}}
+
+subpackages:
+  - range: py-versions
+    name: py${{range.key}}-${{vars.pypi-package}}
+    description: python${{range.key}} version of ${{vars.pypi-package}}
+    dependencies:
+      runtime:
+        - py3-jinja2
+        - py3-yaml
+      provider-priority: ${{range.value}}
+      provides:
+        - py3-${{vars.pypi-package}}
+    pipeline:
+      - name: Python Build
+        uses: python/build-wheel
```

Comment:
Should probably be `py/pip-build-install`

Reply:
RJ, when I try to use this pipeline then I'm not able to build for python-3.10.

```bash
2024/11/08 12:47:49 WARN   creating '/tmp/pip-modern-metadata-cbsh28yi/ansible_core-2.18.0.dist-info' subpackage=py3.10-ansible-core uses=py/pip-build-install name="pip build"
2024/11/08 12:47:49 INFO   Preparing metadata (pyproject.toml): finished with status 'done' subpackage=py3.10-ansible-core uses=py/pip-build-install name="pip build"
2024/11/08 12:47:49 INFO INFO: pip is looking at multiple versions of ansible-core to determine which version is compatible with other requirements. This could take a while. subpackage=py3.10-ansible-core uses=py/pip-build-install name="pip build"
2024/11/08 12:47:49 WARN ERROR: Package 'ansible-core' requires a different Python: 3.10.15 not in '>=3.11' subpackage=py3.10-ansible-core uses=py/pip-build-install name="pip build"
```

Reply:
We can't build with 3.10 as it needs at least 3.11 so let's remove 3.10